temp.py

```diff
@@ -0,0 +1,8 @@
+def reset_files(filepath, file_type="txt"):
+    with open(filepath, "w") as output_file:
```

Comment:
Add a check to ensure that the `filepath` provided is a valid path and that the file exists before attempting to open it. [medium]